cfme/tests/automate/test_git_import.py

```diff
@@ -45,7 +45,8 @@ def new_user(appliance):
         value_assign="Database",
     )
     yield user
-    user.delete_if_exists()
+    user = appliance.rest_api.collections.users.get(name=user.name)
+    user.action.delete()
```

Comment:
@ganeshhubale check exists with rest and then delete... but the reference of BZ as well

Reply:
Actually I am keeping these changes as it was (```user.delete_if_exists()```) [this issue](https://bugzilla.redhat.com/show_bug.cgi?id=1720273) got fixed in 5.11.0.11